File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/specificationService.js
// 品牌服务层
app.service('specificationService',function($http){
	
	// 条件分页查询
	this.search = function(page,rows,searchEntity){
		return $http.post("../specification/search.do?pageNum="+page+"&pageSize="+rows,searchEntity);
	}
	
	// 保存
	this.add = function(entity){
		return $http.post("../specification/add.do",entity);
	}
	
	// 规格查询
	this.selectOptionList = function(){
		return $http.get('../specification/selectOptionList.do');
	}
})

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.grouppojo.Specification;
import com.pinyougou.pojo.TbSpecification;

import entity.PageResult;

public interface SpecificationService {

	PageResult findPage(TbSpecification specification, Integer pageNum, Integer pageSize);

	void add(Specification specification);

	List<Map> selectOptionList();

}
